main.go

```diff
@@ -47,6 +48,7 @@ func init() {
 	flags.BoolVar(&relative, "tag-relative", false, "file paths should be relative to the directory containing the tag file.")
 	flags.BoolVar(&listLangs, "list-languages", false, "list supported languages.")
 	flags.StringVar(&fields, "fields", "", "include selected extension fields (only +l).")
+	flags.StringVar(&extraSymbols, "extra", "", "Add receiver and module  information (+q) in symbol name")
```

Comment:
Change the help text to `include additional tags with package and receiver name prefixes (+q) `.